web/src/main/java/se/inera/intyg/webcert/web/service/facade/impl/GetCertificatesAvailableFunctionsImpl.java

```diff
@@ -412,6 +401,49 @@ public GetCertificatesAvailableFunctionsImpl(AuthoritiesHelper authoritiesHelper
         return resourceLinks;
     }
 
+    private String getRenewBody(Certificate certificate) {
+        if (isLisjpCertificate(certificate)) {
+            final var complementaryText =
+                isUserAndCertificateFromSameCareUnit(certificate) ? "Eventuell kompletteringsbegäran kommer att klarmarkeras."
```

Comment:
Would be great if the logic deciding the message, for renewal, is covered by unit tests.

Reply:
The check for same or different care unit seems to be based on **unit or care unit** (whichever is present in valdVardEnhet and unit). Apparently there are some confusion in requirements regarding unit vs care unit, but requirements says the check should be on care unit. So, perhaps have another look at whether the check should be strictly on care unit or not...

If strictly care unit there might be useable code in PR 14151

Reply:
Good catch @mikeyOne !